### PR TITLE
Features/230306 export data

### PR DESCRIPTION
**Pull request by philipkcl**
* https://github.com/culturesofknowledge/emlo-project/issues/184

### CODE REVIEW

File: person/views.py [hunk 1]
```diff
@@ -422,62 +424,59 @@ def query_fieldset_list(self) -> Iterable:
...
-            "Related resources",
             "Mentioned",
-            "Editor's notes",
+            "Person or Group ID",
+            "Editors' notes",
             "Further reading",
             "Images",
-            "Change user",
+            "Other details",
+            "Other details",
```
Comment (J4bbi) [inline person/views.py:455]: Other details is repeated 2x

File: core/helper/view_utils.py [hunk 2]
```diff
@@ -241,44 +281,46 @@ def get_context_data(self, **kwargs):
...
                 log.exception(e)
 
-        # create csv file and send email in other process
-        Process(target=_fn).start()
+        # create file and send email in other thread
+        Thread(target=_fn).start()
 
         # stay as same page
-        self.to_user_messages = ['Csv file will be send to your email later.']
+        self.to_user_messages = ['File will be send to your email later.']
```
Comment (J4bbi) [inline core/helper/view_utils.py:301]: "File will be send to your email later." => "The selected data is being processed and will be sent to your email soon."

File: core/templates/core/basic_search_page.html [hunk 3]
```diff
@@ -129,10 +144,23 @@ <h3 class="sr-only">Layout</h3>
...
+            let vname_action_url = "{% url return_quick_init_vname '__entry_id__' %}";
         {% endif %}
         let recref_mode = "{{ recref_mode }}";
 
+        $('#export_excel_btn').click(function (e){
+            let msg = 'Are you sure export Excel?'
+            let num_records = $('#num_records').val()
+            if (num_records) {
+                msg = 'Are you sure export ' + num_records + ' records in Excel?'
+            }
```
Comment (J4bbi) [inline core/templates/core/basic_search_page.html:156]: 'Are you sure export Excel?' => 'Are you sure you want to export the search results to Excel?'

'Are you sure export ' + num_records + ' records in Excel? '=> 'Are you sure you want to export {num_records} search results to Excel?'

**Review by J4bbi — CHANGES_REQUESTED** [review]
There will be some html and/or stuff that needs parsing in some fields from `CofkUnionQueryableWork`, such as images and manifestations, but these are not massive issues and can be resolved going forward.

File: core/export_data/excel_header_values.py [hunk 4]
```diff
@@ -0,0 +1,333 @@
...
+class PersonExcelHeaderValues(HeaderValues):
+
+    def get_header_list(self) -> list[str]:
+        return [
+            "EMLO Person ID",
+            "Person primary name in EMLO",
+            "Synonyms",
+            "Roles/Titles",
+            "Gender",
+            "Is Organization (Y=yes;black=no)",
```
[the remark this answers is not in the document]
Comment (philipkcl) [reply core/export_data/excel_header_values.py:157]: header copy from old excel, should be correct

**Pull request merged**